Watchman Revamp

Gyrandola opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->

## Watchman Revamp

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

- New armor and armor sprites for the citywatch, credits to INFRAREDBARON
![image](https://github.com/user-attachments/assets/609b0758-854e-4136-b8b9-a065b3d36255)


- Cudgel changed to Stun Maces (ATTENTION: ITS ONLY A RESKIN, NO ACTUAL STUN EFFECTS)
- Disable stunmaces from being purchaseable.

- Watchman Outfit
<details>

<summary>Details</summary>

![image](https://github.com/user-attachments/assets/d52474b5-36c1-4824-90ef-1e823c03ba73)
![image](https://github.com/user-attachments/assets/c9949f1a-4d84-400d-ba5e-69505f7977aa)

</details>


- Sheriff Outfit (He wears a full cape instead of the half one)
<details>

<summary>Details</summary>

![image](https://github.com/user-attachments/assets/abb3be37-49df-4d02-af6a-ee727e1c9792)
![image](https://github.com/user-attachments/assets/4cf9d00f-7557-4702-bc56-a1c3734581c1)

</details>

Nakalon commented:
omg this is so sexy?

Nakalon commented on code/game/objects/items/rogueweapons/melee/special.dm:
```diff
@@ -132,17 +142,7 @@
-			if(user.a_intent)
-				var/datum/intent/I = user.a_intent
-				if(istype(I))
-					I.afterchange()
```
wait so stun maces lost their stun entirely for more damage?

Gyrandola replied:
they have the same values of cudgels now. Its a temporary measure while I think of less overpowered stun effects

Nakalon replied:
its just cuz I know noble people like to use these to troll ingame

NikolayByondPlayer commented:
Somehow I get Terry Pratchett vibes

Nakalon commented:
> Somehow I get Terry Pratchett vibes 

Omg Niko absolutely. I so would like to go full diskworld and make dwarves genderless

Keazra commented:
Can we have access to the armory like the boggies have access to theirs?

Gyrandola commented:
Removed some mineable area near the town, added lucratice mining area beyond the Bastion to incentivize traffic. 
Added a secondary tunnel that passes through the mountain (still beyond the bastion).
Should have fixed the runtimes

Nakalon reviewed (approved):
Tried and tested
